src/char/inter.c

```diff
@@ -524,45 +524,53 @@ void mapif_accinfo_ack(bool success, int map_fd, int u_fd, int u_aid, int accoun
  **/
 void inter_savereg(uint32 account_id, uint32 char_id, const char *key, unsigned int index, intptr_t val, bool is_string)
 {
-	if( key[0] == '#' && key[1] == '#' ) { // global account reg
+	char esc_val[254];
+	char esc_key[32];
+
+	
```

Comment:
Remove one newline here please. It actually looks like you wanted to put it after 534(where it would fit better).